sympy/solvers/solvers.py

```diff
@@ -2296,8 +2296,13 @@ def solve_linear_system(system, *symbols, **flags):
     {}
 
     """
+    do_simplify = flags.get('simplify', False)
+
     assert system.shape[1] == len(symbols) + 1
 
+    if do_simplify:
+        system = system.applyfunc(simplify)
```

Comment:
Thinking about it I don't really see the point of the `simplify` flag here. The user can call simplify if they want before calling `solve_linear_system`.

Reply:
> Thinking about it I don't really see the point of the `simplify` flag here. The user can call simplify if they want before calling `solve_linear_system`.

So, should I remove it?

Reply:
Yes, I think so